packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -561,25 +568,58 @@ export abstract class FluidDataStoreContext
 		this.channel!.setConnectionState(connected, clientId);
 	}
 
-	public process(
-		message: ISequencedDocumentMessage,
+	/**
+	 * Process the messages to maintain backwards compatibility. The `processMessages` function is added to
+	 * the channel in 2.4.0. For channels before that, call `process` for each message.
+	 */
+	private processMessagesCompat(
+		channel: IFluidDataStoreChannel,
+		message: ISequencedRuntimeMessageCore,
+		messageContents: IRuntimeMessageContents[],
+		local: boolean,
```

Comment:
similar feedback as in DDS PR - consider grouping last 3 args into its own type / single argument - logically it's one thing, and it's a bit easier to follow. Maybe other way to say - the parts do not make much sense on their own, they always come together.

Reply:
Updated it.